e2e/app.e2e.ts

```diff
@@ -194,5 +235,110 @@ describe('bterm launch', function() {
       .then(() => appToMinimize.isRunning())
       .then(result => expect(result).to.be.false);
   });
+
+   it('should increase the window height and width maximize', () => {
+    return this.app.client.waitUntilWindowLoaded()
+      .then(() => this.app.client.browserWindow.getBounds())
+      .then((bounds) => {
+        expect(bounds.width).to.equal(600);
+        expect(bounds.height).to.equal(460);
+      })
+      .then(() => wait(1000))
+      .then(() => this.app.client.click('.maximize'))
+      .then(() => wait(1000))
+      .then(() => this.app.client.browserWindow.getBounds())
+      .then((bounds) => {
+        expect(bounds.width).to.not.equal(600);
+        expect(bounds.height).to.not.equal(460);
+      })
+      .then(() => wait(1000))
+      .then(() => {
+        if (process.platform !== 'darwin') {
+          this.app.client.browserWindow.isMaximized()
+          .then((result) => expect(result).to.be.true);
+        } else {
+          this.app.client.browserWindow.isFullScreen()
+          .then((result) => expect(result).to.be.true);
+        }
+      });
+  });
+
+  it('should decrease the window height and width minimize', () => {
+    let width = 0;
+    let height = 0;
+    return this.app.client.waitUntilWindowLoaded()
+      .then(() => this.app.client.click('.maximize'))
+      .then(() => wait(1000))
+      .then(() => this.app.client.browserWindow.getBounds())
+      .then((bounds) => {
+        width = bounds.width;
+        height = bounds.height;
+      })
+      .then(() => this.app.client.click('.maximize'))
+      .then(() => wait(1000))
+      .then(() => this.app.client.browserWindow.getBounds())
+      .then((bounds) => {
+        expect(bounds.width).to.not.equal(width);
+        expect(bounds.height).to.not.equal(height);
+      });
+  });
+
+  it('should set browserWindow.setBounds() correctly', () => {
```

Comment:
this seems to be that you are testing some electron functionallity and not the application?